common/helpers/move/get-update-links.js

```diff
@@ -0,0 +1,45 @@
+const i18n = require('../../../config/i18n')
+
+const getUpdateUrls = require('./get-update-urls')
+
+const getUpdateLink = (category, urls) => {
```

Comment:
I don't see any explicit tests for `getUpdateLink` - only `getUpdateLinks`

Reply:
Because we're only interested in what `getUpdateLinks` returns. `getUpdateLink` is just an internal method.